polar-core/src/vm.rs

```diff
@@ -2944,6 +3026,23 @@ impl Runnable for PolarVirtualMachine {
                 .collect();
         }
 
+        self.log_with(
+            LogLevel::Info,
+            || {
+                if bindings.is_empty() {
+                    "RESULT: SUCCESS".to_string()
+                } else {
+                    let mut out = "RESULT: { ".to_string(); // open with single right-padded curly
+                    for (key, value) in &bindings {
+                        out.push_str(&format!("{}: {} ", key, value)); // write right-padded key: value pairs
+                    }
+                    out.push('}'); // closing curly
+                    out
```

Comment:
Probably best to put each binding on a new line since the expressions can get pretty long